src/cli.js

```diff
@@ -198,6 +202,27 @@ const cli = async (args) => {
                 backupScheduler(isDebug);
             }
         }
+        if(options.restore)
+        {
+            //restore database from the backup;
+            if(configObj.dbSetupComplete) {
+                let restoreConfig = await restore.prepareRestore();
+                let restoreSetup;
+                if(restoreConfig != null) {
+                restoreSetup = await db.dbRestore(restoreConfig.backupFileName);
+                }
+            }
+            else {
+                console.log('Finish the db configuration below before restoring from the backup');
```

Comment:
we won't get a backup immediatly after db config, so we cannot try restoring if there is no db config at all.